openassessment/xblock/static/js/src/studio/oa_edit.js

```diff
@@ -38,7 +38,7 @@ export class StudioView {
     this.fixModalHeight();
 
     // Initializes the tabbing functionality and activates the last used.
-    this.initializeTabs();
+    this.tabsWidget = this.initializeTabs();
```

Comment:
This doesn't return anything. Beside it is probably better just to assign attribute to some value within that function.